src/app/organization/loan-provisioning-criteria/view-loan-provisioning-criteria/view-loan-provisioning-criteria.component.ts

```diff
@@ -0,0 +1,86 @@
+/** Angular Imports */
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatDialog } from '@angular/material';
+import { Router, ActivatedRoute } from '@angular/router';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatTableDataSource } from '@angular/material/table';
+
+/** Custom Services */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/** Dialog Component */
+import { DeleteDialogComponent } from 'app/shared/delete-dialog/delete-dialog.component';
+
+/**
+ * View Loan Provisioning
+ */
+@Component({
+  selector: 'mifosx-view-loan-provisioning-criteria',
+  templateUrl: './view-loan-provisioning-criteria.component.html',
+  styleUrls: ['./view-loan-provisioning-criteria.component.scss']
+})
+export class ViewLoanProvisioningCriteriaComponent implements OnInit {
+
+  /** Loan Provisioning data. */
+  provisioningData: any;
+  /** Loan Product String. */
+  loanProducts = '';
+  /** Column Headers for Mat Table. */
+  displayedColumns: string[] = ['category', 'minAge', 'maxAge', 'percentage', 'liabilityAccount', 'expenseAccount'];
+  /** Data source for loan provisioning criteria table. */
+  dataSource: MatTableDataSource<any>;
```

Comment:
```suggestion
  dataSource: MatTableDataSource<any>;
  
```

Reply:
Kept the blank space. Please take a look